experimental-design/README.adoc

```diff
@@ -141,5 +141,70 @@ Some search engines as MaxQuant use the Fraction Group to perform better statist
 - Comment [Fraction Group]: Some Quantitative tools use the Fraction Group to know which fractions belong together. In MaxQuant the Fraction Group is called "Experiment".
 
 
+[[encoding-MSRun-technical-details]]
+== MSRun technical details properties
 
+We RECOMMEND to encode some of the technical parameters of the mass spectrometry experiment as Comments (https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html[Check what is a Comment in SDRF]) including the following parameters:
 
+- Protein Modifications <<encoding-protein-modifications>>
+- Precursor and Fragment mass tolerances <<encoding-tolerances>>
+- Enzyme <<encoding-enzymes>>
```

Comment:
Change to Restriction Enzyme, if any. Some searches like for MHC peptides don't require any enzyme.
some searches use multiple enzymes. some search results are from multiple proteases.

Reply:
This section is still not added or written. We need to discuss this one after we decide the PTMs section.